core/input/input_event.h

```diff
@@ -547,6 +548,9 @@ class InputEventMIDI : public InputEvent {
 	void set_controller_value(const int p_controller_value);
 	int get_controller_value() const;
 
+	void set_connection_index(const int p_connection_index);
```

Comment:
```suggestion
	void set_connection_index(int p_connection_index);
```
While it's used elsewhere here we don't use `const` for simple types so let's not use it for new methods